block_cascade/executors/databricks/resource.py

```diff
@@ -123,4 +123,7 @@ def __post_init__(self):
         if self.group_name is None:
             self.group_name = os.environ.get("DATABRICKS_GROUP", "default-group")
         if self.storage_location is None:
-            self.storage_location = f"s3://lakehouse-data-{self.environment}/{self.group_name}/{self.environment}/cascade/"
+            if self.environment == 'prod':
+                self.storage_location = f"s3://lakehouse-data-production/{self.group_name}/cascade/"
+            else:
+                self.storage_location = f"s3://lakehouse-data-staging/{self.group_name}/{self.environment}/cascade/"
```

Comment:
Since I now understand there's a closed source and an open source, should this only be exposed in the closed source ? 

From a usability perspective, `storage_location` is a required parameter not `Optional` because as an outside user, I have zero permission to use this bucket.

Feel free to merge since this received security approval just wanted to raise this for visibility.

Reply:
➕

Reply:
Yeah, I think we can remove. It will require some broader refactoring, but it makes sense.